### ISSUE TITLE
Include "Provider number" in GetDataHolderBrands Response

### ISSUE THREAD

**Issue by seanke** [ConsumerDataStandardsAustralia/standards-maintenance#635]
# Description

When looking up a Dataholder on the cdr.govt.au website we can see a "Provider number" given to each Dataholder. 

![image](https://github.com/ConsumerDataStandardsAustralia/standards-maintenance/assets/34672648/44cec9ab-72a6-4dd3-bd11-d089876e06d6)

Currently, when calling GetDataHolderBrands, a list of RegisterDataHolderBrand objects are returned. This object has a child object called LegalEntityDetail. This child object looks like this
```
{
  "legalEntityId": "string",
  "legalEntityName": "string",
  "logoUri": "string",
  "registrationNumber": "string",
  "registrationDate": "2019-10-24",
  "registeredCountry": "string",
  "abn": "string",
  "acn": "string",
  "arbn": "string",
  "industryCode": "string",
  "organisationType": "SOLE_TRADER"
}
```

The "Provider number" is not one of the values returned.

# Change Proposed
Can "Provider number" please be added to the LegalEntityDetail object.

**Comment by nils-work** [comment]
@perlboy - I think you referred to a field that was missing from Register endpoints in the last MI call, was it this one?

**Comment by perlboy** [comment]
> [@perlboy](https://github.com/perlboy) - I think you referred to a field that was missing from Register endpoints in the last MI call, was it this one?

Yep but in [#679 that's the one](https://github.com/ConsumerDataStandardsAustralia/standards-maintenance/issues/679#issuecomment-2733302040) that comes to mind specifically for Recipients where it's buried in `RegisterDataRecipient` as `accreditationNumber` and in the DCR it's referred via indirection of `legalEntityId`. 

My guess of the table structure is the foreign key and the associated "accreditation number" is shared with "legalEntityRef" for linking, along with a parallel UUID but I don't think that's usable on cdr.gov.au. I'm pretty sure it's used equally on both Holder and Recipient within RAAP as a reference for the underlying Legal Entity but obfuscated at a presentation level (i.e. internal refs are the DHBNK/DH/ADRBNK/ADR provider references but auth systems are keyed off UUID).

@JamesMBligh For Holder links like this one: https://www.cdr.gov.au/find-a-provider?provider=DH777810. Useful for dashboards because otherwise you're stuck with some other unpublished id like this one https://www.cdr.gov.au/find-a-provider?provider=000418
